src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractFeaturesEngine.java

```diff
@@ -86,8 +92,18 @@ public ExtractFeaturesEngine(final String projectID,
         this.variantContextMerger = new ReferenceConfidenceVariantContextMerger(annotationEngine, vcfHeader);
 
     }
+
+    // taken from GnarlyGenotypingEngine
+    private final static double INDEL_QUAL_THRESHOLD = GenotypeCalculationArgumentCollection.DEFAULT_STANDARD_CONFIDENCE_FOR_CALLING - 10 * Math.log10(HomoSapiensConstants.INDEL_HETEROZYGOSITY);
+    private final static double SNP_QUAL_THRESHOLD = GenotypeCalculationArgumentCollection.DEFAULT_STANDARD_CONFIDENCE_FOR_CALLING - 10 * Math.log10(HomoSapiensConstants.SNP_HETEROZYGOSITY);
+
```

Comment:
should we think about an abstract superclass for these constants?

Reply:
Agreed depending on which way we go.  If we end up depending on Gnarly in the long term, we can get rid of this.  If we fold Gnarly into this code (as much as we need) we can share this